templates/.gitlab-ci-template.yml

```diff
@@ -49,23 +49,25 @@ variables:
   # https://gitlab.com/gitlab-org/gitlab-runner/-/issues/27384#note_497228752
   before_script: &script-global-docker-wait
     - |
-      if command -v docker &> /dev/null
-      then
+      if command -v docker &> /dev/null; then
          for i in $(seq 1 30)
          do
            docker info && break
            echo "Waiting for docker to start..."
            sleep 1s
          done
       fi
-      if command -v docker-compose &> /dev/null
-      then
+      if command -v docker-compose &> /dev/null; then
         docker-compose version
       fi
-      if command -v gcloud &> /dev/null
-      then
-        gcloud auth configure-docker
+      if command -v gcloud &> /dev/null; then
```

Comment:
should we print here the gcloud cli version ?

Reply:
Added